audio/set_board_prop.te

```diff
@@ -0,0 +1,10 @@
+type set_board_prop, domain;
+type set_board_prop_exec, exec_type, file_type, vendor_file_type;
+init_daemon_domain(set_board_prop)
+
+set_prop(set_board_prop, audio_product_board_prop)
+
+allow set_board_prop vendor_toolbox_exec:file execute_no_trans;
+allow set_board_prop vendor_shell_exec:file rx_file_perms;
+allow set_board_prop vendor_toolbox_exec:file rx_file_perms;
+allow set_board_prop shell_exec:file rx_file_perms;
```

Comment:
remove this line, under treble, it's not proper for a vendor domain to access a core domain.

Reply:
updated